app/api/login+api.ts

```diff
@@ -0,0 +1,104 @@
+import crypto from 'node:crypto';
```

Comment:
@ProchaLu should these auth files `app/api/login+api.ts`, `app/api/logout+api.ts` and `app/api/register+api.ts` be in side of `app/(auth)` as originally planned?

eg. right now:

```
app/api/login+api.ts
app/api/logout+api.ts
app/api/register+api.ts
```

Should they instead be here?

```
app/(auth)/api/login+api.ts
app/(auth)/api/logout+api.ts
app/(auth)/api/register+api.ts
```

That would better match [the Next.js Fall 2024 example repo](https://github.com/upleveled/next-js-example-fall-2024-atvie/tree/main/app):

![Screenshot 2024-11-02 at 22 33 59](https://github.com/user-attachments/assets/1ff44ce1-815a-4426-8a26-623b68c203dc)

Reply:
Changed to `(auth)/api/..`
[5e63643](https://github.com/upleveled/expo-example-fall-2024-atvie/pull/1/commits/5e6364303ce95173da41d6c805c49c913293dead)